impeller/entity/entity_unittests.cc

```diff
@@ -1696,7 +1697,7 @@ TEST_P(EntityTest, RuntimeEffect) {
   ASSERT_TRUE(runtime_stage->IsDirty());
 
   bool expect_dirty = true;
-  const Pipeline<PipelineDescriptor>* first_pipeline;
+  const raw_ptr<Pipeline<PipelineDescriptor>*> first_pipeline;
```

Comment:
This doesn't look right, you have a pointer to a pointer here.  It should be `raw_ptr<PipelineDescriptor>>`, no?

Reply:
This is an outdated diff...